tests/test_translate.py

```diff
@@ -135,3 +136,29 @@ def test_translate_dont_remove_extra_files(tmpdir):
 
     assert len(list(output.iterdir())) == 6
     assert "extra.txt" in [str(x.name) for x in output.iterdir()]
+
+
+def test_external_translator(tmpdir):
+    tmpdir = Path(str(tmpdir))
+
+    def external_translator(all_objects, output_dir, options):
+        assert {"foo": "bar", "allow_reserved": True} == options
+
+        for category in all_objects:
+            with (tmpdir / category).open('w') as fd:
+                for metric in category:
+                    fd.write("{}\n".format(metric))
+
+    translate.translate_metrics(
```

Comment:
Not for this PR, if you don't have time, but it would be very nice to have a simple example like this one in the docs :-)